src/plugins/intel_cpu/src/node.cpp

```diff
@@ -554,6 +554,7 @@ std::vector<memory::format_tag> Node::getAvailableFormatsForDims(const Shape &di
 void Node::updateShapes() {
     IE_ASSERT(isDynamicNode()) << "Node::updateShapes() is called to a static shape node of type: " << getTypeStr() << " with name: " << getName();
     if (needShapeInfer()) {
+        PERF_SHAPE_INFER(this);
         auto result = shapeInfer();
         if (ShapeInferStatus::success == result.status) {
             redefineOutputMemory(result.dims);
```

Comment:
It does make sense to measure `redefineOutputMemory` call separatelly from `shapeInfer`. Also, it may be usefull to include `needShapeInfer()` into shape inference time measurements.